src/locales/en.yaml

```
@@ -501,3 +502,56 @@ app:
       invalid: indicates local changes to the repo
       packages: Packages
       release_notes: Release notes
+  timelapse:
+    btn:
+      render: Render
+      save_frames: Save Frames
+      open_render_settings: Open Render Settings
+    error:
+      newframe: Error taking timelapse frame
+    label:
+      frames: %{frames} frame | %{frames} frames
```

```suggestion
      frames: '%{frames} frame | %{frames} frames'
```

i18n Ally was showing an error in VSCode for this: as this starts with "%", it needs to be quoted to work properly, or use a ">-" line break!